fcrepo-configs/src/main/resources/config/file-s3/repository.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name" : "repo",
+    "jndiName" : "",
+    "workspaces" : {
+        "predefined" : ["default"],
+        "default" : "default",
+        "allowCreation" : true,
+        "cacheSize" : 10000
+    },
+    "storage" : {
+        "persistence": {
+            "type": "file",
+            "path" : "${fcrepo.object.directory:target/objects}"
+        },
+        "binaryStorage" : {
+            "type" : "s3",
+            "username" : "${aws.accessKeyId}",
```

Comment:
Using a separate configuration file with the sensitive info directly in the file is certainly possible now — but I also think it's fine to continue using system properties, which can be convenient for deployments that aren't concerned about hostile users having access to their process list.  Maybe keeping the system properties as-is, but adding a note about the security implications (in the config files themselves and to the wiki where the usage is described) is a good way to address the problem?